banned-club.sql

```diff
@@ -2598,6 +2611,9 @@ VALUES
     ('spic'),
     ('spit on me'),
     ('spouse beater'),
+    ('stalk me'),
+    ('Stalk me plz'),
+    ('stab me')
```

Comment:
The filters need to have all lowercase characters

Reply:
Will change that